charts/keycloak-operator/templates/operator/deployment.yaml

```diff
@@ -3,6 +3,9 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "keycloak-operator.operator.fullname" . }}
+  {{- with .Values.operator.namespace }}
+  namespace: {{ . }}
+  {{- end }}
```

Comment:
Is it also required that you can deploy the operator in another namespace the the one the Helm chart is being deployed into?
I can fully understand that you would want this for the keycloak instances, but the operator would be deployed in a central namespace right?